packages/forms/src/components/Fields/types.ts

```diff
@@ -87,4 +87,8 @@ export interface Props {
     errorMessages?: { [K: string]: any };
     // add error message
     addErrorMessage: (key: string, message: string) => void;
+    /** Show all prefix/suffix/decorator elements. If set to false, just a label-value pair remains */
```

Comment:
Maybe we can just specify all of the affected elements here (prefix, suffix, helper text, and character count).

```suggestion
    /** Show prefix, suffix, helper text, and character count elements. If set to false, only the input and label will be rendered. */
```